andela/wg-wits
# #159236212 Change author field from single field to foreign field

#### What does this PR?
Have the license author field as a foreign field.
#### Description of Task to be completed?
Change the license author field for an exercise from a single field to a foreign field. It should be represented as a foreign key in the exercise table.
#### How should this be manually tested?
**Follow the following commands**.
- `git fetch ch-Change-authors-field-to-foreign-key`
- `git checkout  ch-Change-authors-field-to-foreign-key`
-  _**Drop the database**_
- `python manage.py migrate`
- `invoke load-fixtures`
- **Run the server**
   - ` python manage.py runserver`
- Log into the application with any user and create a new exercise.
- Check the exercise table, the author is referenced as licence_author_id.
#### Screenshots
<img width="922" alt="screen shot 2018-08-29 at 10 46 37" src="https://user-images.githubusercontent.com/39955261/44773280-db642a00-ab78-11e8-914c-79dea87fd770.png">
